### DISCUSSION TITLE
LDM Notes for May 17th, 2023

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#7206] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-05-17.md

## Agenda

- Inline arrays

**Comment by huoyaoyuan** [comment]
> Be more granular with `ref struct` usage in `async` methods.

Don't forget `yield return` generators if you plan to do this. I have met actual usages where `ROS<char>` is passed as parameter instead of string.
